cabal-install/Distribution/Client/ProjectPlanning.hs

```diff
@@ -1522,7 +1522,7 @@ elaborateInstallPlan verbosity platform compiler compilerprogdb pkgConfigDB
             external_exe_map = Map.fromList $
                 [ (getComponentId pkg, path)
                 | pkg <- external_exe_dep_pkgs
-                , Just path <- [planPackageExePath pkg] ]
+                , path <- planPackageExePaths pkg ]
```

Comment:
Hmm I just realized only the first path will be picked, since we convert to a `Map ComponentId FilePath`. Which is a problem if some build tool has multiple exes. I'll change the map to `Map ComponentId [FilePath]`.